docs/developer/extending/apps/developing-apps/app-sdk/apl.md

```diff
@@ -71,19 +71,29 @@ import { APL, AuthData } from "@saleor/app-sdk/apl";
 const client = createClient();
 await client.connect();
 
+// If you intend to use the same Redis instance for multiple Apps,
+// add prefix to the keys you will keep AuthData:
```

Comment:
**polish**: Something is missing in this sentence, please consider rewriting it.